utils/risks.json

```diff
@@ -641,7 +577,6 @@
 				"singlesided",
 				"ssc",
 				"Levered-AAVE-wBTC",
```

Comment:
This should be removed.

Reply:
Missed this one, thank you

Reply:
Fixed with [72d7bad](https://github.com/yearn/yearn-watch/pull/4/commits/72d7bad5d086015db2845615252f60a9cfa4ca3f)